app/views/IndexView.scala.html

```diff
@@ -175,8 +175,6 @@
 
 @viewOrAmendReturnsLink = @{link(id = "view-submitted-returns-link", text = messages("account.homePage.card.history.viewreturnslink"), call = routes.SubmittedReturnsController.onPageLoad)}
 
-@paymentsHistoryLink = @{link(id = "payment-history-link", text = messages("account.homePage.card.history.viewpaymentslink"), call = routes.IndexController.onPageLoad)}
```

Comment:
Should the message key/values also be removed?

Reply:
yes, that was removed.